ambari-server/src/main/resources/stacks/BIGTOP/3.2.0/repos/repoinfo.xml

```diff
@@ -23,4 +23,11 @@
       <reponame>bigtop</reponame>
     </repo>
   </os>
+  <os family="redhat8">
```

Comment:
make this change in BIGTOP 3.3.0 also, seems this repoinfo is not inheritted in higher stack version

Reply:
> make this change in BIGTOP 3.3.0 also, seems this repoinfo is not inheritted in higher stack version

resolved